_posts/2020-03-15-learning-to-share-accomplishments-1.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Learning to Share Accomplishments: Part 1 - How to Share Accomplishments"
+date: 2020-03-15T09:00:00-04:00
+permalink: /2020/03/15/learning-to-share-accomplishments-1/
+author_profile: true
+toc: true
+toc_label: "Post Contents"
+toc_icon: "file-alt"
+medium_post:
+  - ""
+categories:
+  - Accomplishments
+  - Job Hunting
+tags:
+  - 
+---
+
+This post is a part of a series on job hunting and job interviewing.
+{: .notice--info}
+
+<figure>
+    <img src='https://www.hdwallpaper.nu/wp-content/uploads/2017/04/cat-11.jpg' alt='Until I figure out a better image' width="50%" height="50%" />
+    <figcaption>Until I figure out a better image to put here.....</figcaption>
+</figure>
+
+In [my last post](/2019/09/02/reflections-on-the-latest-round-of-job-hunting/), I talked about my last round of job 
+hunting. More specifically, I mention one of the pieces of feedback was on how I come off as a bit more of an 
+"individual contributor" (someone who just takes tasks and does them) as opposed to someone who actually designs or 
+engineers solutions (someone who takes a vague thing and maps how how it will work, breaks it apart into smaller 
+tasks, then can work on some of those). 
+
+After some thought, I realized this is because over time I've learned to give credit to those on my team for some of 
+the things we've worked on. I think it feels better to have a team take credit (well, and take on the failures) of all 
+aspects of a project. While this is important and I think every team should do this, it sucks for a technical job 
+interviews. It doesn't work to "sell yourself" to a future employer.
+
+For part 1 of this post, I want to spend some time thinking through how to talk about accomplishments. What's the right 
+way to do it without feeling like I have an inflated ego? How do I own up to my work, even when I've been on teams?
+Why does it feel so hard to do?
+
+## "Bragging" About Yourself
+
+As someone who now has been in dozens of conferences, podcasts, videos, websites, blogs/articles, etc., I've had to 
+come to terms with tooting my own horn some. All of these things usually want a bio from me. It's a lot easier now that 
+I have made multiples and keep them around for things like this. In the beginning, it wasn't easy. In fact it was just 
+downright terrible. It just felt wrong almost to try to write about all these things. It felt like bragging.
+
+The easiest way I found to toot my own horn? I found one rule makes it easy: **Stick to the facts.**
```

Comment:
I want to find a way to break up this "block" of text.
Maybe "Stick to the facts" should become a heading? Maybe an image/gif could be added around here?

Reply:
Hmm... I could maybe do that...